src/modules/MainPageActions/MainPageActions.module.css

```diff
@@ -0,0 +1,40 @@
+@import '_colors.css';
+@import '_fonts.css';
+@import '_sizes.css';
+
+.main-page-actions {
+  padding: 1rem;
+
+  &__heading {
+    margin-bottom: 1rem;
+
+    @mixin main-block-header;
+  }
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__button {
+    flex: 1 1 auto;
+    width: auto;
+    max-width: calc(25% - 1rem);
+
+    @media (max-width: $screenMobileMax) {
+      width: 100%;
+      max-width: 100%;
+      margin-bottom: 1rem;
+    }
+  }
+
+  &__left-column {
```

Comment:
we can create mixin for column styles

Reply:
I think  one __colum is enough. Will change.